samples/boards/cc3220sf_launchxl/simplelink_shell/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _simplelink_shell:
+
+SimpleLink WLAN Shell for CC3220SF WiFi
+########################################
+
+Overview
+********
+
+This sample enables testing of the WiFi offload management operations
+of the SimpleLink WiFi driver, and shows how to request and get
+notifications of WLAN events using the Zephyr network management
+framework, :file:`doc/subsystems/networking/network-management-api.rst`.
```

Comment:
change to create a link using the label found at the top of the .rst file:

    framework, :ref:`network_management_api`.